x/concentrated-liquidity/genesis_test.go

```diff
@@ -167,6 +275,42 @@ func (s *KeeperTestSuite) TestInitGenesis() {
 				},
 			},
 			expectedPositions: []model.Position{testPositionModel, positionWithPoolId(testPositionModel, 2)},
+			expectedAccumValues: []genesis.AccumObject{
+				{
+					Name:        "fee/1",
+					Value:       sdk.NewDecCoins(sdk.NewDecCoin("foo", sdk.NewInt(10))),
+					TotalShares: sdk.NewDec(10),
+				},
+				{
+					Name:        "fee/2",
+					Value:       sdk.NewDecCoins(sdk.NewDecCoin("bar", sdk.NewInt(20))),
+					TotalShares: sdk.NewDec(20),
+				},
+			},
```

Comment:
What about incentives accumulators? Are they stored together with fee accumulators? Can we have a test where there are incentives accumulators as well

Reply:
ditto below